packages/web/components/Dashboard/LatestContentTabs/Read.tsx

```diff
@@ -1,18 +1,15 @@
 import { Box, Heading, Link, LoadingState, Text } from '@metafam/ds';
-import React, { useEffect, useState } from 'react';
-import { parse } from 'rss-to-json';
+import React from 'react';
 import useSWR from 'swr';
 
-export const Read: React.FC = () => {
-  const [url, setUrl] = useState<string | null>(null);
-  const { data, error } = useSWR(url, parse);
-
-  useEffect(() => setUrl(`${window.location.origin}/metagame/feed`), []);
+const fetcher = (url: string) => fetch(url).then((res) => res.json());
 
+export const Read: React.FC = () => {
+  const { data, error } = useSWR('/api/feed', fetcher);
   return (
     <Box pt={4}>
-      {data ? (
-        data.items?.map((item) => (
+      {data && data.response ? (
+        data.response.items?.map((item: any) => (
```

Comment:
For this sort of thing you can define a type with just the properties being used (`title`) to avoid this warning

Reply:
Done